sdk/python/setup.py

```diff
@@ -138,6 +139,6 @@ def find_version(*file_path_parts):
         'console_scripts': [
             'dsl-compile = kfp.deprecated.compiler.main:main',
             'dsl-compile-v2 = kfp.compiler.main:main',
-            'kfp=kfp.__main__:main'
```

Comment:
**Not this line of code**

Copyright 2022 here?

Reply:
Done.